src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Machine started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Machines stopped working.");

```

Comment:
The stop work message should also be more informative and indicate that it is the `Bulldozer` that has stopped working.